backend/database/migrations/2025_04_09_235929_create_employees_table.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('employees', function (Blueprint $table) {
+            $table->id();
+            $table->string('first_name');
+            $table->string('middle_name')->nullable();
+            $table->string('last_name');
+            $table->string('suffix')->nullable();
+            $table->enum('gender', ['male', 'female', 'other'])->nullable();
+            $table->date('birthdate');
+            $table->enum('civil_status', [
+                'single',
+                'married',
+                'widowed',
+                'divorced',
+                'separated',
+                'anulled',
+                // 'cohabiting', should we add cohabiting/live-in?
+            ]);
```

Comment:
1. Create Enum Class:
```
// app/enums/CivilStatus

class CivilStatus {
case Single = 'single'
...
} 
```

```suggestion
            $table->enum('civil_status', array_column(CivilStatus:cases(), 'value'));
```